src/lib.rs

```diff
@@ -4,6 +4,3 @@ pub mod contract_info;
 pub mod error;
 pub mod msg;
 pub mod state;
-
-#[cfg(target_arch = "wasm32")]
-cosmwasm_std::create_entry_points_with_migration!(contract);
```

Comment:
Instead of automagically creating the endpoints with this call, cosmwasm wants those `entry_point` annotations.  This function no longer exists in 1.0.0